src/util.js

```diff
@@ -320,13 +328,8 @@ module.exports = {
       return;
     }
 
-    if (typeof window !== 'undefined') {
-      if (window.crypto) {
-        return window.crypto.subtle || window.crypto.webkitSubtle;
-      }
-      if (window.msCrypto) {
-        return window.msCrypto.subtle;
-      }
+    if (typeof window !== 'undefined' && window.crypto) {
+      return window.crypto.subtle || window.crypto.webkitSubtle;
```

Comment:
Please rebase on top of the current master, as this change would revert IE11 support.